<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EtriMainWindow</class>
 <widget class="QMainWindow" name="EtriMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electre Tri</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="right_layout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="group_input">
        <property name="title">
         <string>Input Layer</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QComboBox" name="combo_layer"/>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="Bloadlayer">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_profiles">
        <property name="title">
         <string>Profiles</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCommandLinkButton" name="Badd_profile">
           <property name="text">
            <string>Add Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="Bdel_profile">
           <property name="text">
            <string>Del Profile</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_thresholds">
        <property name="maximumSize">
         <size>
          <width>388</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Thresholds</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="5" column="0">
          <widget class="QCheckBox" name="cbox_samethresholds">
           <property name="text">
            <string>Use same for all profiles</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="cbox_noveto">
           <property name="text">
            <string>No Veto</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="cbox_sameqp">
           <property name="text">
            <string>Indifference = Preference</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_affectation">
        <property name="title">
         <string>Affectation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="cutlevel_layout">
           <item>
            <widget class="QLabel" name="label_cutlevel">
             <property name="text">
              <string>Cutting level:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinbox_cutlevel">
             <property name="minimum">
              <double>0.500000000000000</double>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>0.750000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="procedure_layout">
           <item>
            <widget class="QLabel" name="label_procedure">
             <property name="text">
              <string>Procedure:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_procedure">
             <item>
              <property name="text">
               <string>Pessimist</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Optimist</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="Bgenerate">
        <property name="text">
         <string>Generate Decision Map</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="Tab_params">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_criterions">
       <attribute name="title">
        <string>Criterions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="table_crit">
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string>Criteria</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Weight</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_profiles">
       <attribute name="title">
        <string>Profiles</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableWidget" name="table_prof"/>
        </item>
        <item row="1" column="0">
         <widget class="QTabWidget" name="tab_thresholds">
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_indiff">
           <attribute name="title">
            <string>Indifference</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1">
             <widget class="QTableWidget" name="table_indiff"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_pref">
           <attribute name="title">
            <string>Preference</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_pref"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_veto">
           <attribute name="title">
            <string>Veto</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QTableWidget" name="table_veto"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
